test/sandbox/sandbox.go

```diff
@@ -193,25 +194,31 @@ func (s S) Git() Git {
 	return s.git
 }
 
-// Loads metadata for sandbox
-func (s S) LoadMetadata() terramate.Metadata {
+// Generate generates code for all stacks on the sandbox
+func (s S) Generate() {
 	s.t.Helper()
 
-	meta, err := terramate.LoadMetadata(s.RootDir())
+	err := generate.Do(s.RootDir(), s.RootDir())
 	assert.NoError(s.t, err)
-	return meta
 }
 
-// Generate generates code for all stacks on the sandbox
-func (s S) Generate() {
+// LoadStacks load all stacks from sandbox rootdir.
+func (s S) LoadStacks() []stack.S {
```

Comment:
Ah we even have a LoadStacks, so maybe also having a LoadStack for a single stack makes sense (one of the tests seemed like could benefit from it).